backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -10,35 +11,87 @@
 from datetime import datetime
 from typing import Dict, List, Set, Tuple, Union
 
+from sqlalchemy import Connection
+
 DB_DIR = os.path.dirname(os.path.realpath(__file__))
 BACKEND_DIR = os.path.join(DB_DIR, "..")
 PROJECT_ROOT = os.path.join(BACKEND_DIR, "..")
 sys.path.insert(0, str(PROJECT_ROOT))
 from backend.utils import call_github_action
 from backend.db.resolve_fetch_failures_excess_items import resolve_fetch_failures_excess_items
 from backend.db.utils import SCHEMA, fetch_status_set_success, get_db_connection, reset_temp_refresh_tables, \
-    select_failed_fetches, refresh_derived_tables, sql_in, sql_query
+    run_sql, select_failed_fetches, refresh_derived_tables, sql_in, sql_query
 from enclave_wrangler.objects_api import concept_set_members__from_csets_and_members_to_db, \
-    fetch_cset_and_member_objects, get_csets_over_threshold
+    fetch_cset_and_member_objects, fetch_cset_version, get_csets_over_threshold
 
 DESC = "Resolve any failures resulting from fetching data from the Enclave's objects API."
 
 
 def _report_success(
-    cset_ids: List[int], fetch_audit_row_lookup: Dict[int, Dict], comment_addition: str = None, use_local_db=False
+    cset_ids: Union[int, List[int]], fetch_audit_row_lookup: Dict[int, Dict], comment_addition: str = None, use_local_db=False
 ):
-    """Report success for a list of concept set IDs.
+    """Report success fetch success for given concept sets.
     todo: kind of weird to pass a lookup. Maybe do that beforehand and pass row dict"""
+    cset_ids = [cset_ids] if not isinstance(cset_ids, list) else cset_ids
     success_rows = []
     for cset_id in cset_ids:
         row = fetch_audit_row_lookup[cset_id]
         if comment_addition:
             row['comment'] = row['comment'] + '; ' + \
-            'Success result: ' if not comment_addition.startswith('Success result: ') else '' + comment_addition
+                 ('Success result: ' if not comment_addition.startswith('Success result: ') else '') + comment_addition
         success_rows.append(row)
     fetch_status_set_success(success_rows, use_local_db)
 
 
+def _ad_hoc_detect_and_delete_discarded_drafts(schema: str = SCHEMA, local: bool = False):
```

Comment:
## Manually deleted all discarded drafts
Not every discarded draft was in `fetch_audit`. I queried all drafts in the DB, then fetched to see which were discarded, and deleted from the DB.

They (n=16) were:
869225473, 44040290, 106884385, 426891141, 202217606, 247241223, 301311463, 835748713, 241060906, 432161229, 739057725, 133725107, 36874682, 197574523, 522429, 563361919

FYI, I backed up the `code_sets` rows for these as JSON and uploaded to [Google Drive](https://drive.google.com/drive/folders/1QOKWJGe_8LyDFgEcYTBcafc0GHD2J9fj).